src/contexts/ThemeContext.js

```diff
@@ -0,0 +1,5 @@
+import { createContext } from 'react';
+
+const ThemeContext = createContext('light');
```

Comment:
Pls always use CONSTANTS for string values and especially those you are using several times through the app

Reply:
fixed